src/context/WalletProvider/config.ts

```diff
@@ -73,7 +73,10 @@ import { XDEFIFailure } from './XDEFI/components/Failure'
 import { XDEFIConfig } from './XDEFI/config'
 
 export interface SupportedWalletInfo {
-  adapters: any[]
+  adapters: {
+    // TODO(gomes): can we type this?
```

Comment:
Probably not, same as before

Reply:
i think we can, if we declare a base interface which is used as a basis for each implementation
`loadAdapter: () => Promise<IWalletBaseAdapter>`

Reply:
there seems to be `GenericAdapter` already which can be a starting point